### PR TITLE
HDDS-10152. Field should be static

### PR DESCRIPTION
**Pull request by sarvekshayr**
## What changes were proposed in this pull request?
Changing the fields mentioned in the HDDS-10152.txt from final to static final. Renaming the variables and its subsequent appearance to ALL_CAPS format.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-10152

## How was this patch tested?
Patch was tested by running the test class and the corresponding test class of server-side class to ensure that the changes made ran successfully without throwing an error.

### CODE REVIEW

File: hadoop-ozone/fault-injection-test/mini-chaos-tests/src/test/java/org/apache/hadoop/ozone/MiniOzoneChaosCluster.java [hunk 1]
```diff
@@ -385,7 +385,7 @@ public Set<DatanodeDetails> dnToFail() {
   }
   
   @Override
-  public void restartHddsDatanode(DatanodeDetails dn, boolean waitForDatanode)
+  public void restartHddsDatanode(DatanodeDetails dn, boolean Datanode)
```
Comment (adoroszlai) [inline hadoop-ozone/fault-injection-test/mini-chaos-tests/src/test/java/org/apache/hadoop/ozone/MiniOzoneChaosCluster.java:388]: This looks like unintentional change.

**Review by adoroszlai — COMMENTED** [review]
Thanks @sarvekshayr for working on this.

* Please remove files under `hadoop-ozone/integration-test/container.db`, they are leftover after running some tests.
* There are some checkstyle failures, see results at https://github.com/sarvekshayr/ozone/actions/runs/7783793573/job/21222995834

File: hadoop-ozone/fault-injection-test/mini-chaos-tests/src/test/java/org/apache/hadoop/ozone/MiniOzoneChaosCluster.java (hunk 1 above)
Comment (sarvekshayr) [reply hadoop-ozone/fault-injection-test/mini-chaos-tests/src/test/java/org/apache/hadoop/ozone/MiniOzoneChaosCluster.java]: Changes have been addressed and committed.

**Review by tanvipenumudy — APPROVED** [review]
Thank you @sarvekshayr for the patch! 
LGTM +1, pending Git CI.

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java [hunk 2]
```diff
@@ -142,18 +142,18 @@ private void createPrincipals() throws Exception {
...
     createPrincipal(this.adminKeytabInOtherHost, adminPrincipalInOtherHost);
 
-    this.userPrincipal1 = this.user1 + "/" + hostAndRealm;
-    this.userPrincipal2 = this.user2 + "/" + hostAndRealm;
-    this.userKeytab1  = new File(workDir, this.user1 + ".keytab");
-    this.userKeytab2  = new File(workDir, this.user2 + ".keytab");
+    this.userPrincipal1 = this.USER_1 + "/" + hostAndRealm;
+    this.userPrincipal2 = this.USER_2 + "/" + hostAndRealm;
+    this.userKeytab1  = new File(workDir, this.USER_1 + ".keytab");
+    this.userKeytab2  = new File(workDir, this.USER_2 + ".keytab");
```
Comment (adoroszlai) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java:156]: Better not access `static` via instance (`this`):

```suggestion
    this.userPrincipal1 = USER_1 + "/" + hostAndRealm;
    this.userPrincipal2 = USER_2 + "/" + hostAndRealm;
    this.userKeytab1  = new File(workDir, USER_1 + ".keytab");
    this.userKeytab2  = new File(workDir, USER_2 + ".keytab");
```

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java [hunk 3]
```diff
@@ -313,40 +313,40 @@ private static void doAs(UserGroupInformation ugi,
   public void testListVolumeWithOtherUsersListAllAllowed() throws Exception {
     setupEnvironment(true, true);
 
-    // Login as user1, list other users' volumes
+    // Login as USER_1, list other users' volumes
```
Comment (adoroszlai) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java:316]:
```suggestion
    // Login as user1, list other users' volumes
```

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java [hunk 4]
```diff
@@ -313,40 +313,40 @@ private static void doAs(UserGroupInformation ugi,
...
           "volume5"), true);
-      checkUser(adminUser, Arrays
+      checkUser(ADMIN_USER, Arrays
           .asList("volume1", "volume2", "volume3", "volume4", "volume5",
               "volume6", "s3v"), true);
       return true;
     });
 
-    // Login as user2, list other users' volumes
+    // Login as USER_2, list other users' volumes
```
Comment (adoroszlai) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java:326]:
```suggestion
    // Login as user2, list other users' volumes
```

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java [hunk 5]
```diff
@@ -361,39 +361,39 @@ public void testListVolumeWithOtherUsersListAllAllowed() throws Exception {
   public void testListVolumeWithOtherUsersListAllDisallowed() throws Exception {
     setupEnvironment(true, false);
 
-    // Login as user1, list other users' volumes, expect failure
+    // Login as USER_1, list other users' volumes, expect failure
```
Comment (adoroszlai) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java:364]:
```suggestion
    // Login as user1, list other users' volumes, expect failure
```

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java [hunk 6]
```diff
@@ -361,39 +361,39 @@ public void testListVolumeWithOtherUsersListAllAllowed() throws Exception {
...
+      checkUser(USER_2, Arrays.asList("volume2", "volume3", "volume4",
           "volume5"), false);
-      checkUser(adminUser, Arrays.asList("volume1", "volume2", "volume3",
+      checkUser(ADMIN_USER, Arrays.asList("volume1", "volume2", "volume3",
               "volume4", "volume5", "volume6", "s3v"), false);
       return true;
     });
 
-    // Login as user2, list other users' volumes, expect failure
+    // Login as USER_2, list other users' volumes, expect failure
```
Comment (adoroszlai) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java:373]:
```suggestion
    // Login as user2, list other users' volumes, expect failure
```

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java [hunk 7]
```diff
@@ -502,28 +502,28 @@ public void testAclDisabledListAllAllowed() throws Exception {
...
+    // Login as USER_1, list their own volumes
     doAs(userUGI1, () -> {
-      checkUser(user1, Arrays.asList("volume1", "volume3", "volume5"),
+      checkUser(USER_1, Arrays.asList("volume1", "volume3", "volume5"),
           true);
       return true;
     });
 
-    // Login as user2, list their own volumes
+    // Login as USER_2, list their own volumes
```
Comment (adoroszlai) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java:512]:
```suggestion
    // Login as user2, list their own volumes
```

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java [hunk 8]
```diff
@@ -502,28 +502,28 @@ public void testAclDisabledListAllAllowed() throws Exception {
   public void testAclDisabledListAllDisallowed() throws Exception {
     setupEnvironment(false, false);
 
-    // Login as user1, list their own volumes
+    // Login as USER_1, list their own volumes
```
Comment (adoroszlai) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java:505]:
```suggestion
    // Login as user1, list their own volumes
```

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java [hunk 9]
```diff
@@ -471,28 +471,28 @@ public void testAclEnabledListAllDisallowed() throws Exception {
   public void testAclDisabledListAllAllowed() throws Exception {
     setupEnvironment(false, true);
 
-      // Login as user1, list their own volumes
+      // Login as USER_1, list their own volumes
```
Comment (adoroszlai) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java:474]:
```suggestion
      // Login as user1, list their own volumes
```

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java [hunk 10]
```diff
@@ -471,28 +471,28 @@ public void testAclEnabledListAllDisallowed() throws Exception {
...
     doAs(userUGI1, () -> {
-      checkUser(user1, Arrays.asList("volume1", "volume3", "volume5"),
+      checkUser(USER_1, Arrays.asList("volume1", "volume3", "volume5"),
           true);
       return true;
     });
 
-    // Login as user2, list their own volumes
+    // Login as USER_2, list their own volumes
```
Comment (adoroszlai) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java:481]:
```suggestion
    // Login as user2, list their own volumes
```

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java [hunk 11]
```diff
@@ -436,14 +436,14 @@ public void testAclEnabledListAllAllowed() throws Exception {
   public void testAclEnabledListAllDisallowed() throws Exception {
     setupEnvironment(true, false);
 
-    // Login as user1, list their own volumes
+    // Login as USER_1, list their own volumes
```
Comment (adoroszlai) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java:439]:
```suggestion
    // Login as user1, list their own volumes
```

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java [hunk 12]
```diff
@@ -403,30 +403,30 @@ public void testListVolumeWithOtherUsersListAllDisallowed() throws Exception {
   public void testAclEnabledListAllAllowed() throws Exception {
     setupEnvironment(true, true);
 
-    // Login as user1, list their own volumes
+    // Login as USER_1, list their own volumes
```
Comment (adoroszlai) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java:406]:
```suggestion
    // Login as user1, list their own volumes
```

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java [hunk 13]
```diff
@@ -403,30 +403,30 @@ public void testListVolumeWithOtherUsersListAllDisallowed() throws Exception {
...
     doAs(userUGI1, () -> {
-      checkUser(user1, Arrays.asList("volume1", "volume3", "volume4",
+      checkUser(USER_1, Arrays.asList("volume1", "volume3", "volume4",
           "volume5"), true);
       return true;
     });
 
-    // Login as user2, list their own volumes
+    // Login as USER_2, list their own volumes
```
Comment (adoroszlai) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerListVolumesSecure.java:413]:
```suggestion
    // Login as user2, list their own volumes
```

**Review by adoroszlai — COMMENTED** [review]
Thanks @sarvekshayr for updating the patch.

Noticed some very minor issues:
1. avoid access to `static` via `this`
2. let's keep "user1", "user2" in comments instead of using constant's name, since that's the actual value

**Review by adoroszlai — APPROVED** [review]

**Comment by adoroszlai** [comment]
Thanks @sarvekshayr for the patch, @tanvipenumudy for the review.

**Pull request merged**